index.js

```diff
@@ -84,26 +85,35 @@ cli.on('respawn', function(flags, child) {
 });
 
 function run() {
-  cli.launch({
+  var envOpts = {
     cwd: opts.cwd,
     configPath: opts.gulpfile,
     require: opts.require,
     completion: opts.completion,
-  }, handleArguments);
+  };
+
+  cli.launch(envOpts, function(env) {
+    var config;
+    try {
+      config = loadConfigFiles(env.configFiles['.gulp'], ['home', 'cwd']);
```

Comment:
This exception is caused by validation of config files. Does your comment mean that outputting error and exiting should be done in `loadConfigFiles`, or it is unnecessary to validate config files?